tests/unit/test_dags.py

```diff
@@ -95,4 +95,5 @@ def test_pipelines_dags():
         'ReturnToOfficeBookingsPipeline',
         'StaffSSOUsersPipeline',
         'TeamsDatasetPipeline',
+        'TagsClassifierPipeline',
```

Comment:
Super tiny, but it would be nice to keep this in alphabetical order... :)
```suggestion
        'TagsClassifierPipeline',
        'TeamsDatasetPipeline',
```